firefox_media_tests/manifest.ini

```diff
@@ -1 +1,3 @@
-[include:playback/manifest.ini]
+[include:youtube/manifest.ini]
+[include:eme/manifest.ini]
```

Comment:
Let's not include the EME/generic tests by default for now so they have to be specified via command-line options instead. True EME tests won't work without a special profile and url-list, so shouldn't be part of default run anyway. We can come back to this this once the refactoring is done later on.